java/pom.xml

```diff
@@ -398,7 +398,14 @@
         </plugin>
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>3.0.0-M7</version>
+          <version>2.22.2</version>
```

Comment:
Is there any way to avoid downgrading surefire?

Reply:
@lidavidm 
I believe it's related to https://issues.apache.org/jira/browse/SUREFIRE-2033

Reply:
Hmm. Is it possible to downgrade to only, say, 3.0.0-M4, and add a comment linking to the regression? Or otherwise, is it possible to configure the runners to avoid this?

Reply:
@lidavidm Done. Fixed by adding extra decencies to surefire, the version remains '3.0.0-M7'